docs/user/localization.rst

```diff
@@ -1,17 +1,25 @@
 Localization of Documentation
 =============================
 
-.. note:: This feature only applies to Sphinx documentation. We are working to bring it to our other documentation backends.
+In this article, we explain high-level approaches to internationalizing and localizing your documentation.
 
-Read the Docs supports hosting your docs in multiple languages.
-There are two different things that we support:
+By default, Read the Docs assumes that your documentation is or *might become* multilingual one day.
+The initial default language is English and
+therefore you often see the initial build of your documentation published at ``/en/latest/``,
+where the ``/en`` denotes that it's in English.
+By having the ``en`` URL component present from the beginning,
+you are ready for the eventuality that you would want a second language.
```

Comment:
This is a great intro 👍

Reply:
Thanks :blush: 

I hope we can have some time to expand fresh ideas in this section, but I think we should do it in another iteration. Will mark for later :+1: